### ISSUE TITLE
Pause and resume commands use HTTP POST instead of PUT

### ISSUE THREAD

**Issue by 56quarters** [datamountaineer/kafka-connect-tools#21]
Hi there,

First off, great project, thanks so much for creating it.

I noticed that pause and resume commands are using `POST` when talking to the Kafka Connect REST API instead of `PUT`

Specifically, these methods should be using `PUT`:

https://github.com/datamountaineer/kafka-connect-tools/blob/master/src/main/scala/com/datamountaineer/connect/tools/RestKafkaConnectApi.scala#L198

https://github.com/datamountaineer/kafka-connect-tools/blob/master/src/main/scala/com/datamountaineer/connect/tools/RestKafkaConnectApi.scala#L206

This is the result printed from the Connect Worker:

```
[2017-02-08 17:09:54,503] INFO 127.0.0.1 - - [08/Feb/2017:22:09:54 +0000] "POST /connectors/sqlserver-test-sink/pause HTTP/1.1" 405 323  2 (org.apache.kafka.connect.runtime.rest.RestServer:60)
```

Note the HTTP 405 response.

The API for these is specified here:

http://docs.confluent.io/3.1.2/connect/restapi.html#put--connectors-(string-name)-pause

I can submit a PR for these changes if you'd like. Thanks!

**Comment by andrewstevenson** [comment]
@56quarters Great, PRs welcome!

**Comment by andrewstevenson** [comment]
@56quarters Do you want to cover #20 aswell?

**Comment by 56quarters** [comment]
I can submit a PR for this issue and I'll see what I can do for #20. I'll do them as separate commits / PRs if I make any progress on the latter.

**Comment by 56quarters** [comment]
I decided to only fix #21, not #20. I think #20 requires some judgement about the best way to do that and didn't feel like I could make that decision. Thanks!
